website/public/js/controllers/inventoryCtrl.js

```diff
@@ -152,12 +160,16 @@ habitrpg.controller("InventoryCtrl",
         $scope.selectedFood = null;
 
         _updateDropAnimalCount(user.items);
+        if ($scope.mountCount > startingMounts && !user.preferences.suppressModals.raisePet) {
```

Comment:
Using `mountCount` prevents the modal being shown when quest pets are raised to mounts (mountCount is for tracking Mount Master progress). Elsewhere, `statCalc.totalCount(profile.items.mounts)` is used for counting all mounts.